google-ads-examples/src/main/java/com/google/ads/googleads/examples/planning/GetCampaignCriterionBidModifierSimulations.java

```diff
@@ -0,0 +1,136 @@
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @param campaignId the campaign ID to get the criterion bid modifier simulations.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    try (GoogleAdsServiceClient googleAdsServiceClient =
+        googleAdsClient.getLatestVersion().createGoogleAdsServiceClient()) {
+      // Creates a query that retrieves the criterion bid modifier simulations.
+      String query =
+          String.format(
+              "SELECT campaign_criterion_simulation.criterion_id, "
+                  + "campaign_criterion_simulation.start_date, "
+                  + "campaign_criterion_simulation.end_date, "
+                  + "campaign_criterion_simulation.bid_modifier_point_list.points "
+                  + "FROM campaign_criterion_simulation "
+                  + "WHERE campaign_criterion_simulation.type = BID_MODIFIER "
+                  + "AND campaign_criterion_simulation.campaign_id = %d",
+              campaignId);
+      // Constructs the SearchGoogleAdsStreamRequest.
+      SearchGoogleAdsStreamRequest request =
+          SearchGoogleAdsStreamRequest.newBuilder()
+              .setCustomerId(Long.toString(customerId))
+              .setQuery(query)
+              .build();
+
+      // Creates and issues a search Google Ads stream request.
+      ServerStream<SearchGoogleAdsStreamResponse> stream =
+          googleAdsServiceClient.searchStreamCallable().call(request);
+
+      // Iterates over all rows in all messages and prints the requested field values for
+      // the campaign criterion bid modifier simulation in each row.
+      for (SearchGoogleAdsStreamResponse response : stream) {
+        for (GoogleAdsRow googleAdsRow : response.getResultsList()) {
+          CampaignCriterionSimulation simulation = googleAdsRow.getCampaignCriterionSimulation();
+          System.out.printf(
+              "Found campaign-level criterion bid modifier simulation for "
+                  + "criterion with ID %d, start date '%s', end date '%s', and points:%s'",
+              simulation.getCriterionId().getValue(),
+              simulation.getStartDate().getValue(),
+              simulation.getEndDate().getValue());
+        }
+      }
```

Comment:
The loop that prints the points list is missing, see the [golden](https://github.com/googleads/google-ads-php/blob/master/examples/Planning/GetCampaignCriterionBidModifierSimulations.php#L140) for reference.